### PR TITLE
Write hotfix procedure doc

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The frontend for Codex, written in React with Material UI components. This proje
 
 Found a bug? Submit a report [here](https://github.com/WildMeOrg/codex-frontend/issues/new).
 
-Developer contributions are very much appreciated. Refer to the [contribution guide](https://github.com/WildMeOrg/codex-frontend/blob/master/CONTRIBUTION_GUIDE.md). If you are a Python dev looking to help with our project, take a look at the [backend](https://github.com/WildMeOrg/houston). If you are a data scientist looking to help, take a look at [WBIA](https://github.com/WildMeOrg/wildbook-ia).
+Developer contributions are very much appreciated. Refer to the [contribution guide](https://github.com/WildMeOrg/codex-frontend/blob/master/docs/contribution-guide.md). If you are a Python dev looking to help with our project, take a look at the [backend](https://github.com/WildMeOrg/houston). If you are a data scientist looking to help, take a look at [WBIA](https://github.com/WildMeOrg/wildbook-ia).
```
Comment (Emily-Ke) [inline README.md:9]: Should `master` be `develop` in the URL?

File: docs/hotfix-procedure.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Next you need to fix the bug. Unfortunately this guide can't tell you how to fix the bug. After that though:
+
+5. Create a pull request to merge your changes into the `main` branch.
+6. Two code reviews are required for hotfixes. Reviewers, be careful, you will probably be the final set of eyes before this goes to production.
+7. Merge the pull request 
+8. Create a new tag for your release and deploy to production. 
+
+Finally, you need to get your change into the develop branch as well.
+
+9. `git pull origin develop`
```
Comment (Atticus29) [inline docs/hotfix-procedure.md:19]: Wait.... we shouldn't be pulling FROM develop onto main or a main-adjacent branch.

Instead, I propose here:
1. git checkout develop
2. git pull origin develop (good habit)
3. git checkout -b new-develop-offshoot
4. git merge nameOfBranchFromStep3Section1
5. Resolve merge conflicts.
6. Push
7. Issue PR into develop.

Comment (brmscheiner) [reply docs/hotfix-procedure.md:19]: I think this is just a shortcut for those steps? If you are on your hotfix branch you will be one commit ahead of main and many commits behind develop. So when you pull develop you will initiate the merge of develop into your hotfix

File: docs/contribution-guide.md [hunk 3]
```diff
@@ -30,3 +30,7 @@ If you want to help translate the project, that is very much appreciated and nee
 - Components that are reused widely go in `/src/components`
 - Post any questions in Github issues or send an email to ben@wildme.org 
 - Thanks for contributing =)
+
+## Making a hotfix
+
+Likely you wont need to make a hotfix unless you are a Wild Me employee, but if the need arises please follow the [hotfix procedures](https://github.com/WildMeOrg/codex-frontend/blob/master/docs/hotfix-procedure.md).
```
Comment (Emily-Ke) [inline docs/contribution-guide.md:36]: `master` should be `develop` here too.

File: docs/hotfix-procedure.md (hunk 2 above)
Comment (Atticus29) [reply docs/hotfix-procedure.md:19]: Ah ok. That's fine, then. Just gives me the heebie-jeebies.

**Review by Atticus29 — APPROVED** [review]

File: docs/hotfix-procedure.md (hunk 2 above)
Comment (brmscheiner) [reply docs/hotfix-procedure.md:19]: I'm gonna rewrite it a little to make it very clear what branch you should be on when you run these commands

**Pull request merged**